flexget/plugins/daemon/irc.py

```diff
@@ -667,6 +673,13 @@ def irc_update_config(manager):
         stop_irc(manager)
         return
 
+    if irc_bot is None:
+        log.error('ImportError: irc_bot module not found. Shutting down daemon.')
+        stop_irc(manager)
+        from flexget.manager import manager
```

Comment:
It should shutdown imo. If it reaches that if, that means the user has specified irc but failed to install the dep. Since it's a daemon plugin it makes more sense to shut down the daemon.